sdk/core/switcher/entry.S

```diff
@@ -315,6 +324,14 @@ exception_entry_asm:
 	reloadRegisters c1, cgp, c4, c5, c6, c7, c8, c9, c10, c11, c12, c13, c14, c15, csp
 	mret
 
+.Linvalid_csp_permissions:
+	cjal 			.Lpop_trusted_stack_frame
+	csh zero, 0(cnull)
```

Comment:
This will trap. Was it committed by mistake? If not how will the subsequent instructions be reached?

Reply:
hehe yes, thanks!